### DISCUSSION TITLE
[Feature request] Extend a monadic idiom u&.(m&{) to dyadic valence

### DISCUSSION THREAD

**Discussion by jip** [jsoftware/jsource#239] [Ideas]
An idiom `u&.(m&{)` is monadic only implementing:
```j
m}&y u m&{ y           NB. u&.(m&{) y
```
I propose to extend it to support dyadic application implementing:
```j
m}&y (m&{ x) u m&{ y    NB. x u&.(m&{) y
```

That is, to make an idiom `u&.(m&{)` ambivalent.

**Comment by HenryHRich** [comment]
Not a bad thought, but the vast majority of uses would not need (m{x) so it would be better to use x instead.  This is a semidual rather than a dual.

This is structural under, which is already a separate case of &. using its own inverse.  Perhaps the dyad case of structural under should be a semidual.
